app/pages/project/index.cjsx

```diff
@@ -27,164 +35,222 @@ SOCIAL_ICONS =
-              </Link>}
-            {if @props.project.redirect
-              <a target="_blank" href={@redirect_classify_link(@props.project.redirect)} className="tabbed-content-tab">
-                <Translate content="project.nav.classify" />
-              </a>
-            else
-              <Link to="#{projectPath}/classify" activeClassName="active" className="classify tabbed-content-tab">
-                <Translate content="project.nav.classify" />
-              </Link>}
-            {unless @props.project.redirect
-              <PromiseRenderer promise={@props.project.get 'pages'}>{(pages) =>
-                pageTitles = @getPageTitles(pages)
-                <span>
-                  {if pageTitles.results
-                    <Link to="#{projectPath}/results" activeClassName="active"className="tabbed-content-tab">
-                      {pageTitles.results}
-                    </Link>}
-                  {if pageTitles.faq
-                    <Link to="#{projectPath}/faq" activeClassName="active" className="tabbed-content-tab">
-                      {pageTitles.faq}
-                    </Link>}
-                  {if pageTitles.education
-                    <Link to="#{projectPath}/education" activeClassName="active" className="tabbed-content-tab">
-                      {pageTitles.education}
-                    </Link>}
-                </span>
-              }</PromiseRenderer>}
-            <Link to="#{projectPath}/talk" activeClassName="active" className="tabbed-content-tab">
-              <Translate content="project.nav.talk" />
-            </Link>
-            {for link, i in @props.project.urls
-              link._key ?= Math.random()
-              {label} = link
-              unless label
-                for pattern, icon of SOCIAL_ICONS
-                  if link.url.indexOf(pattern) isnt -1
-                    socialIcon = icon
-                socialIcon ?= 'globe'
-                label = <i className="fa fa-#{socialIcon} fa-fw fa-2x"></i>
-              <a key={link._key} href={link.url} className="tabbed-content-tab" target="#{@props.project.id}-#{i}">{label}</a>}
-          </nav>
-
-          {if @props.project.configuration?.announcement
-            <div className="informational project-announcement-banner">
-              <Markdown>{@props.project.configuration.announcement}</Markdown>
-            </div>}
-
-          {React.cloneElement(@props.children, {owner: owner, project: @props.project, user: @props.user})}
-
-          {unless @props.project.launch_approved or @props.project.beta_approved
-            <Translate className="project-disclaimer" content="project.disclaimer" component="p" />}
-
-          <PotentialFieldGuide project={@props.project} />
-        </div>
-      }</PromiseRenderer>
-    }</ChangeListener>
-
-module.exports = React.createClass
-  displayName: 'ProjectPageWrapper'
-  mixins: [TitleMixin, HandlePropChanges]
```

Comment:
Are we deprecating `HandlePropChanges`? I was starting to try that out in the workflow assignment feature I'm working on.